common/src/main/java/org/vivecraft/client/gui/settings/GuiRenderOpticsSettings.java

```diff
@@ -24,7 +25,8 @@ public class GuiRenderOpticsSettings extends GuiVROptionsBase {
         VRSettings.VrOptions.HANDHELD_CAMERA_RENDER_SCALE,
         VRSettings.VrOptions.HANDHELD_CAMERA_FOV,
         VRSettings.VrOptions.RELOAD_EXTERNAL_CAMERA,
-        VRSettings.VrOptions.MIRROR_EYE
+        VRSettings.VrOptions.MIRROR_EYE,
+        VRSettings.VrOptions.STEREOPLUGIN
```

Comment:
don't think we have button space for a new setting there

Reply:
there was space on the screen, but I'm not sure where would be the best place

Reply:
iirc all slots are taken by something in the differnt mirror modes

Reply:
that also shouldn't be really in renderting settings, since it's not speciffic to rendering, but also inputs and such

Reply:
We could place it in the main menu instead, but then we also need to move some stuff

Reply:
for my dev stuff I put it in the modmenu/forg mod config screen for now